index.html

```diff
@@ -844,11 +817,15 @@ <h4>Characters</h4>
         </section>
 
         <section>
-          <h4>Shared properties</h4>
+          <h4>Shared properties and Value Sets</h4>
           <p>Some of the properties in the DAPT data model are common within more than one object type,
             and carry the same semantic everywhere they occur.
             These <dfn>shared properties</dfn> are listed in this section.
           </p>
+          <p>Some of the value sets in DAPT are reused across more than one property,
+            and have the same constraints everywhere they occur.
```

Comment:
Represents and Script Represents don't really have put the same constraints on content-descriptor ... Maybe we should remove the end of the sentence.

Reply:
There's no difference to the constraints on `<content-descriptor>`, there's a difference in how many content descriptors are permitted in Script Represents vs Represents, but that difference isn't described in this section.